app/infrastructure/messaging/redis_pubsub.py

```
@@ -278,6 +279,50 @@ async def subscribe(self, tenant_id: str) -> AsyncIterator[SyncProgressEvent]:
             logger.info("Unsubscribed from %s", channel)
 
 
+async def run_sync_progress_broadcast(app: Any) -> None:
```

Ruff (0.15.0)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `app`

(ANN401)